### ISSUE TITLE
Bugfix: Bomberman 64: The Second Attack! Crashes after splashscreen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [facelift/mupen64gc#227]
```
What steps will reproduce the problem?
1. Load Bomberman 64: The Second Attack!

What is the expected output? What do you see instead?
Game loads with a fade from black to white (probably to the Hudson splash 
screen), then after a 
short period of time, it fades back to black, then crashes to a Wii Exception 
error that can reset 
back to the Homebrew Channel

What version of the product are you using? On which system and loader?
Wii64 Beta 1.1 "Honey" through Homebrew Channel 1.0.5
```

Original issue reported on code.google.com by `ironsoni...@gmail.com` on 4 Mar 2010 at 12:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `emuki...@gmail.com` on 11 May 2010 at 12:32
- Changed title: **Bugfix: Bomberman 64: The Second Attack! Crashes after splashscreen**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in > v1.1
```

Original comment by `emuki...@gmail.com` on 9 Oct 2012 at 11:46
- Changed state: **Fixed**
